test/modules/layers/core-layers.spec.js

```diff
@@ -102,48 +102,6 @@ test('ArcLayer', t => {
   t.end();
 });
 
-test('ArcLayer#non-iterable', t => {
```

Comment:
This test targets an edge case in `calculateInstancePositions`, which is removed in this PR.